starfish/.factorypath

```diff
@@ -0,0 +1,66 @@
+<factorypath>
+    <factorypathentry kind="VARJAR" id="M2_REPO/io/quarkus/quarkus-resteasy/1.4.2.Final/quarkus-resteasy-1.4.2.Final.jar" enabled="true" runInBatchMode="false"/>
+    <factorypathentry kind="VARJAR" id="M2_REPO/io/quarkus/quarkus-vertx-http/1.4.2.Final/quarkus-vertx-http-1.4.2.Final.jar" enabled="true" runInBatchMode="false"/>
```

Comment:
This file is not relevant to have in git afaics.

Reply:
@fahad-israr this should be removed. it is eclipse specific and autogenerated.

Reply:
removed @https://github.com/maxandersen/starfish/pull/8/commits/80a33256e3544218d7e9789a008f8c4ad4dba8a6